### PR TITLE
PD-1470 / 25.04 / PD-1470-document-installing-nvidia-drivers

### PR DESCRIPTION
**Pull request by linzi-ix**
Screenshot uploaded of new checkbox option and small description made.
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Review by Simoneu01 — COMMENTED** [review]
In the screenshot, `NVIDIA `is misspelt `NDIVIA`

File: content/TruenasApps/_index.md [hunk 1]
```diff
@@ -235,6 +235,20 @@ If the GPU was previously isolated and/or assigned to a VM, a reboot could be re
 Restart the system then return to the **Resources Configuration** section of the application to see if expected devices are available.
 {{< /expand >}}
 
+## Installing NVIDIA Drivers
+Beginning in TrueNAS 24.10, NVIDIA drivers are no longer automatically installed. Users must instead manually install drivers from the TrueNAS UI.
```
Comment (micjohnson777) [inline content/TruenasApps/_index.md:239]: How about this instead
```suggestion
Beginning in TrueNAS 24.10, NVIDIA drivers are no longer automatically installed. Users must manually install drivers from the TrueNAS UI.
```

File: content/TruenasApps/_index.md [hunk 2]
```diff
@@ -235,6 +235,20 @@ If the GPU was previously isolated and/or assigned to a VM, a reboot could be re
...
+
```
Comment (micjohnson777) [inline content/TruenasApps/_index.md:240]: When writing a procedure, start steps with the verb action. Remember parallel construction principles apply and procedures are about the action. You can add clarification after the action or just add the result of the action. See the suggested changes below.

File: content/TruenasApps/_index.md [hunk 3]
```diff
@@ -235,6 +235,20 @@ If the GPU was previously isolated and/or assigned to a VM, a reboot could be re
...
+1. If running TrueNAS version 24.10 or higher, go to **Apps > Installed** and locate the **Configuration** drop-down menu.
+   
+2. Once you have selected the drop-down menu, click on the **Settings** option to open the **settings configuration panel.**
+   
+3. Users with compatible GPUs now have an **Install NVIDIA Drivers** checkbox available within the configuration panel.
+   
+4. To complete installation, check the **Install NVIDIA Drivers** checkbox, and click **Save.**
```
Comment (micjohnson777) [inline content/TruenasApps/_index.md:247]:
```suggestion
If running TrueNAS 24.10 or higher:
1. Go to **Apps > Installed** and click on the **Configuration**.
   
2. Click on **Settings** to open the **settings configuration panel.**
   
3. Select **Install NVIDIA Drivers**, which is available to users with compatible GPUs.
   
4. Select **Install NVIDIA Drivers**, and click **Save.**
```

File: content/TruenasApps/_index.md [hunk 4]
```diff
@@ -235,6 +235,20 @@ If the GPU was previously isolated and/or assigned to a VM, a reboot could be re
...
+
+{{< trueimage src="/images/SCALE/Apps/AppsInstallNvidiaDrivers.png" alt="Install NVIDIA Drivers" id="Install NVIDIA Drivers" >}}
```
Comment (micjohnson777) [inline content/TruenasApps/_index.md:249]: You can move this image up to the line between step 3 and 4 for better context.

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
A few suggested improvements for clarity and better information flow.

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**